pkg/controller.v1/common/job.go

```diff
@@ -275,7 +275,9 @@ func (jc *JobController) ReconcileJobs(
 					pg = volcanoPodGroup
 					return nil
 				}
-			case GangSchedulerSchedulerPlugins:
+			case GangSchedulerNone:
+				return nil
```

Comment:
Probably, we don't need this condition since we confirm if the JobConfig has a config for gang scheduling in L232.

cc: @alculquicondor

Reply:
Done